### ISSUE TITLE
Missing libwayland dependencies

### ISSUE THREAD

**Issue by johnodon** [luispabon/sway-ubuntu#5]
Am I doing something wrong?  `sudo apt --fix-broken install` got me going again but is this expected?

Vanilla install of latest version of Ubuntu Desktop 21.04. 

```
sudo apt update && sudo apt upgrade -y
sudo apt install make git -y
git clone https://github.com/luispabon/sway-ubuntu.git && cd sway-ubuntu
touch .env
make yolo
```

```
Errors were encountered while processing:
 libwayland-client0:amd64
 libwayland-cursor0:amd64
 libwayland-dev:amd64
 libwayland-server0:amd64
 libwayland-cursor++0:amd64
make: *** [Makefile:202: libwayland-1.19] Error 1
```

**Comment by luispabon** [comment]
Are you running Ubuntu amd64 at all?

**Comment by johnodon** [comment]
> Are you running Ubuntu amd64 at all?

Yes I am.  

https://releases.ubuntu.com/21.04/ubuntu-21.04-desktop-amd64.iso

**Comment by luispabon** [comment]
Thank you. I can see the issue. 

Closed via 7761910648488350e02e73ef2684bc914fb1d5ae
